pkg/rbd/rbd_util.go

```diff
@@ -127,10 +128,16 @@ func createImage(ctx context.Context, pOpts *rbdVolume, volSz int64, cr *util.Cr
 	} else {
 		klog.V(4).Infof(util.Log(ctx, "rbd: create %s size %s format %s using mon %s, pool %s"), image, volSzMiB, pOpts.ImageFormat, pOpts.Monitors, pOpts.Pool)
 	}
+	if pOpts.DataPool != "" {
```

Comment:
(nit) instead of 2 messages when an image is created against an EC pool, can this be folded into one of the messages above?

Reply:
done